Runigram.java

```diff
@@ -122,8 +199,14 @@ public static Color blend(Color c1, Color c2, double alpha) {
 	 * The two images must have the same dimensions.
 	 */
 	public static Color[][] blend(Color[][] image1, Color[][] image2, double alpha) {
-		//// Replace the following statement with your code
-		return null;
+		Color[][] blendImg = new Color[image1.length][image1[0].length];
+		for(int i = 0; i < image1.length; i++){
+			for(int j = 0; j < image1[0].length; j++){
+				Color blend = blend(image1[i][j], image2[i][j], alpha);
+				blendImg[i][j] = blend;

```

Comment:
```suggestion
				blendImg[i][j] = blend(image1[i][j], image2[i][j], alpha);

```

And specifically don't use method names as variables.